src/Components/Blazor/Build/test/BuildIntegrationTests/PublishIntegrationTest.cs

```diff
@@ -155,31 +155,23 @@ public async Task Publish_HostedApp_Works()
             // Make sure the main project exists
             Assert.FileExists(result, publishDirectory, "blazorhosted.dll");
 
-            var blazorPublishDirectory = Path.Combine(publishDirectory, "standalone");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "blazor.boot.json");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "blazor.webassembly.js");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "wasm", "mono.wasm");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "wasm", "mono.js");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "_bin", "standalone.dll");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "_bin", "Microsoft.Extensions.Logging.Abstractions.dll"); // Verify dependencies are part of the output.
+            var blazorPublishDirectory = Path.Combine(publishDirectory, "wwwroot");
+            Assert.FileExists(result, blazorPublishDirectory, "_framework", "blazor.boot.json");
+            Assert.FileExists(result, blazorPublishDirectory, "_framework", "blazor.webassembly.js");
+            Assert.FileExists(result, blazorPublishDirectory, "_framework", "wasm", "mono.wasm");
+            Assert.FileExists(result, blazorPublishDirectory, "_framework", "wasm", "mono.js");
+            Assert.FileExists(result, blazorPublishDirectory, "_framework", "_bin", "standalone.dll");
+            Assert.FileExists(result, blazorPublishDirectory, "_framework", "_bin", "Microsoft.Extensions.Logging.Abstractions.dll"); // Verify dependencies are part of the output.
 
             // Verify static assets are in the publish directory
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "index.html");
+            Assert.FileExists(result, blazorPublishDirectory, "index.html");
```

Comment:
So to clarify, does this PR change the publish output directory structure from `publish/<appname>/dist` to `publish/<appname>/wwwroot`? Fine with me if so, though we should strongly consider @pranavkm's point and whether we could reduce this to just `publish/` by the time we ship in May. Otherwise we may be stuck with this odd directory structure forever.

Reply:
For hosted apps it puts stuff inside the host wwwroot folder.
For standalone apps it keeps them where we were putting them. I'm not trying to address that as part of that PR. I think a cheap and easy solution is to simply publish things to `/publish/wwwroot` in the standalone case too.

I can't see any specific drawback to that.

Reply:
>  I think a cheap and easy solution is to simply publish things to /publish/wwwroot in the standalone case too

Sounds great to me

Reply:
Moving everything inside the wwwroot